trip-ninja-docs/src/App.tsx

```diff
@@ -22,6 +23,7 @@ export default function App() {
           />
         </Switch>
       </Router>
+      <Footer/>
```

Comment:
Would this not display weirdly in the docs pages? Looks like the footer is centered under the content sections vs the entire page

Reply:
hmm right, ok I wrapped the whole thing in a body tag and put the footer under the div with the content sections. is that what you mean?

Reply:
I think its more that it might have to go inside the farestructure component as the sidebar goes all the way to the bottom
<img width="591" alt="Screen Shot 2023-04-14 at 4 31 15 PM" src="https://user-images.githubusercontent.com/104777513/232149201-a85efac4-3f43-4b65-ad97-76e0fe5e6e66.png">